### DISCUSSION TITLE
Install problems :(

### DISCUSSION THREAD

**Discussion by jjgonc** [Budibase/budibase#1952] [Help] [answered]
Hey, after following the self host tutorial on youtube with Linux (https://www.youtube.com/watch?v=2P4HaFkP6Zk) I tried to continue and sign up in the budibase page, but I cant find where to do it... Ths way I'm stuck at the minute 2:45 of this video.
 
I searched on the "Post Install" section and it says "Go to the URL of your new installation to start using budibase!", but I can't reach this URL because I used Linux-Ubuntu **command line** to install it.

Can you help me to continue?
Many thanks!

**Comment by shogunpurple** [comment, marked as answer]
@jjgonc  if you installed it on your local machine, it will be `http://localhost:10000` in your browser.

**Reply by jjgonc** [reply]
So, this way I entered but I'd like to download BudiBase Builder as they use in the video. How can I do it?

**Reply by shogunpurple** [reply]
Ah, I see the problem. That video is outdated. Budibase is no longer an electron application - it's completely web based. You can no longer download the builder in this format. We are planning on updating the videos/tutorials/docs very soon with new content.

**Reply by jjgonc** [reply]
Thank you for the fast answer! 
Please do it, it would be very helpfull!

**Reply by jjgonc** [reply]
Sorry to bother you, but I'm new in this "world" and I have some doubts about the platform usage...
Note: I'm using BudiBase via docker (same way as shown in the video above)

1 - If I'm not mistaken, the deploy lauches the project I am doing. But how can I access it in this new web based app (only found how to do it in older Builder version)?
2 - How can I save the project so that when I turn my pc off and then back on I access it back? Once I turn my pc on again, how do I reconnect back to the server host

I know those can be basic questions, but I didn't manage how to solve them...

**Reply by PGLange** [reply]
hi @jjgonc,
1.) let's try it step for step. Please read https://docs.budibase.com/self-hosting/hosting-methods/docker-compose to setup budibase as dockerized system. Also read https://docs.docker.com/engine/install/linux-postinstall/.
This works stable and is pretty fast - even on slow laptops or virtual machines (used it on a 1 core, 4 GB ram VM). To use https pls read https://docs.budibase.com/self-hosting/advanced-options/reverse-proxy.

you can export and import your apps from within the platform. Look in the apps section, and then klick on the three dots "...", there is the export function. Import is located on the upper right at the apps screen.

2.) when your pc / laptop is restarted, use your docker user and change to the directory where your docker-compose.yaml, envoy.yaml and  hosting.properities live. then execute (as your docker user) docker-compose --env-file hosting.properties up. 
this is an easy beginners way.
 When you can handle this check out the fantastic nodejs budi app. budi is your friend, but pls ensure, that you understand docker-compose first. With the nodejs app updates are super easy: use budi hosting --update.
You will love budi hosting --start and budi hosting --stop. But one task at a time! Take your time, Rome wasn't built in a day ;)

Hope this helps :)
